magma/backend/coreir_transformer.py

```diff
@@ -3,6 +3,7 @@
 import json
 import logging
 import os
+import magma as m
```

Comment:
We should avoid this in internal code. Looks like you need:

```
from ..compile import compile
from ..config import get_compile_dir, set_compile_dir
```

If the problem is circular imports, then we do something like

```python
from .verilog import VerilogCompiler

...

VerilogCompiler(bind_module, f".magma/{bind_module.name}").compile()
```

In fact I think this is preferred anyway.